api/migrations/20210812174148-create-guild.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("guilds", {
+      id: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      prefix: {
+        defaultValue: ".",
+        type: Sequelize.STRING,
+      },
+      opRoles: {
+        defaultValue: [],
+        type: Sequelize.ARRAY(Sequelize.STRING),
+      },
+      levels: {
+        defaultValue: "[]",
+        type: Sequelize.STRING,
+      },
+      invites: {
+        defaultValue: "[]",
+        type: Sequelize.STRING,
+      },
+      threadInactivityTime: {
+        defaultValue: 1000 * 60 * 60 * 24 * 7,
+        type: Sequelize.INTEGER,
+      },
+      moderationServer: {
+        type: Sequelize.STRING,
+      },
+      createdAt: {
+        allowNull: false,
```

Comment:
Should this default to now, the current timestamp?

Reply:
Sequelize automatically manages and adds this fields.  
```By default, Sequelize automatically adds the fields createdAt and updatedAt to every model, using the data type DataTypes.DATE. Those fields are automatically managed as well - whenever you use Sequelize to create or update something, those fields will be set correctly. The createdAt field will contain the timestamp representing the moment of creation, and the updatedAt will contain the timestamp of the latest update.```

Reply:
Oh I see. It's been a while since I looked at sequailize.